tests/assert_type/db/models/test_enums.py

```diff
@@ -0,0 +1,132 @@
+import enum
+from typing import Literal, Optional
```

Comment:
Let's convert this to mypy only test file (`.yml` based), because these tests will once be used by pyright as well.

And since this is a plugin-based feature, it won't be supported by pyright.

Reply:
It's only a handful of cases that are affected by the plugin, for which there are `# pytest: ignore[...]` comments.

The majority is still useful for testing the stubs against pyright.

Reply:
ok!